tests/unit/example.spec.js

```diff
@@ -0,0 +1,19 @@
+import { mount } from "@vue/test-utils";
+import Index from "../../src/resources/app/pages";
+
+// every test file should includes AT LEAST ONE "describe()", "it()", and "expect()"
+describe("displays message", () => {
+  // mount() returns a wrapped Vue component we can interact with
+  it("should renders", () => {
+    // The component to test
```

Comment:
"the component to be tested" or "the to-be-tested component"

Reply:
fixed